lib/parse-args.js

```diff
@@ -83,6 +83,12 @@ function buildYargs (withCommands = false) {
       type: 'boolean',
       describe: 'should temp files be deleted before script execution'
     })
+    .options('all', {
+      default: undefined,
+      type: 'string',
+      describe: 'supplying src directory for --all will cause c8 to consider all src files in the directory when ' +
```

Comment:
I would suggest we model after `c8`, and default to `process.cwd()`, taking into account `exclude`/`include` rules.